tests/integration/dashboard/test_catalogue_views.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+
+from oscar.apps.dashboard.catalogue.views import ProductListView
+from oscar.core.loading import get_model
+from oscar.test.factories import create_product
+from oscar.test.utils import RequestFactory
+
+Product = get_model('catalogue', 'Product')
+
+
+class ProductListViewTestCase(TestCase):
+    def test_searching_child_product_by_title_returns_parent_product(self):
```

Comment:
Please add a test to ensure that we don't return duplicates in the event that parent and child share a title.